src/worker.js

```diff
@@ -208,7 +218,7 @@ this.onmessage = function(e) {
           Atomics.store(HEAPU32, (threadInfoStruct + 0 /*C_STRUCTS.pthread.threadStatus*/ ) >> 2, 1); // Mark the thread as no longer running.
 #if ASSERTIONS
           if (typeof(Module['_emscripten_futex_wake']) !== "function") {
-            err("Thread Initialisation failed.");
+            console.error("Thread Initialisation failed.");
```

Comment:
We should be able to use `err`, it's defined on line 50 (although it ends up calling console.error anyway).

Reply:
Thanks! I saw `out()` failed and assumed `err()` was the same, but you're right, it's not. Fixed.